Sources/Markdown/Base/DirectiveArgument.swift

```diff
@@ -201,7 +201,8 @@ public struct DirectiveArgumentText: Equatable {
             var line = TrimmedLine(untrimmedText[...],
                                    source: range?.lowerBound.source,
                                    lineNumber: range?.lowerBound.line,
-                                   parseIndex: parseIndex)
+                                   parseIndex: parseIndex,
+                                   startParseIndex: untrimmedText.index(parseIndex, offsetBy: 1 - (range?.lowerBound.column ?? 1)))
```

Comment:
Looking at the code again, it seems like there's an existing `lineStartIndex` property on this `LineSegment` type that should always be populated. Did that not work for this purpose?

Reply:
`lineStartIndex` is init as `untrimmedText.startIndex` in BlockDirectParser.

eg.
```
(lldb) po untrimmedText
"hello\n @Options(scope: page"

(lldb) po untrimmedText[lineStartIndex]
"h"

(lldb) po untrimmedText[untrimmedText.index(parseIndex, offsetBy: 1 - (range?.lowerBound.column ?? 1))]
" "
```